src/modules/user-settings/components/editor-picker.tsx

```diff
@@ -12,31 +11,16 @@ interface EditorPickerProps {
 
 export const EditorPicker = ( { value, onChange }: EditorPickerProps ) => {
 	const { __ } = useI18n();
-	const [ installedApps, setInstalledApps ] = useState< {
-		vscode: boolean | null;
-		phpstorm: boolean | null;
-	} >( {
-		vscode: null,
-		phpstorm: null,
-	} );
+	const { data: installedApps } = useGetInstalledAppsQuery();
```

Comment:
```suggestion
	const { installedEditors, uninstalledEditors } = useGetInstalledAppsQuery( undefined, {
		selectFromResult: ( result ) => ( {
			installedEditors: selectInstalledEditors( result.data ),
			uninstalledEditors: selectUninstalledEditors( result.data ),
		} ),
	} );
```
If we follow my suggestion of adding selectors to `src/stores/installed-apps-api.ts`, then we can update this call like so.